nexus/src/updates.rs

```diff
@@ -2,38 +2,39 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+use buf_list::BufList;
+use futures::TryStreamExt;
 use nexus_db_queries::db;
 use omicron_common::update::ArtifactsDocument;
 use std::convert::TryInto;
 
 // TODO(iliana): make async/.await. awslabs/tough#213
```

Comment:
I suppose we can drop this `TODO` as part of this PR!

Reply:
done!